tools/jdk/default_java_toolchain.bzl

```diff
@@ -146,6 +146,18 @@ def default_java_toolchain(name, configuration = DEFAULT_TOOLCHAIN_CONFIGURATION
         **toolchain_args
     )
 
+    native.config_setting(
+        name = name + "_version_setting",
+        values = {"java_language_version": toolchain_args["source_version"]},
+        visibility = ["//visibility:private"],
+    )
+    native.toolchain(
+        name = name + "_toolchain",
```

Comment:
The names this generates are kind of confusing, like `//:toolchain_toolchain` and //java/main:default_toolchain_toolchain`.

We still don't have good terminology around this, but would using `_definition` work? then we'd get `//:toolchain_definition` and `//java/main:default_toolchain_definition`, which is at least less redundant.

Reply:
Thanks. Appending `_definition` sounds much better.